assets.json

```diff
@@ -0,0 +1,40 @@
+{
+  "default": {
+      "languageSpecific": {
+          "device": [],
+          "shared": []
+      },
+      "shared": [
+        "Honey_Hunt_Intro_1.mp4", 
+        "Honey_Hunt_Intro_2.mp4", 
+        "Honey_Hunt_Intro_3.mp4", 
+        "Honey_Hunt_Intro_4.mp4", 
+        "Honey_Hunt_Intro_5.mp4",
+        "Honey_Hunt_Intro_Button_3.mp4", 
+        "Honey_Hunt_Intro_Button_4.mp4", 
+        "Honey_Hunt_End.mp4",
+        "Spanish_Honey_Hunt_Intro_1.mp4", 
+        "Spanish_Honey_Hunt_Intro_2.mp4", 
```

Comment:
Can we also remove the Spanish prefix from the naming scheme? So in the file you would just remove those references and in the bucket rename the Spanish files and move them to es -> shared. The files will have identical names but are separated by folders. This removes the need for conditional logic in the app as i18n together with the utility functions handles loading the proper files.